requirements.txt

```diff
@@ -10,4 +10,5 @@ xgboost
 flask
 pytest
 pyope
+ipykernel
```

Comment:
Why are you adding only this? It's a necessary but not sufficient dependency of Jupyter notebooks. I would either add `notebook` as a dependency, or assume the user knows how to install Jupyter on their own (probably the latter).